.changeset/violet-bugs-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lit/react": patch
+---
+
+Lit/React – Wraps `ref` in a stable callback
```

Comment:
```suggestion
Use a stable callback for passing the forwarded `ref` to the underlying custom element.
```

Reply:
Sorry, didn't see this suggestion but I amended it anyway!

Reply:
Your amendment was much better, thank you!